### ISSUE TITLE
Unwrapping Optional global parameter panics

### ISSUE THREAD

**Issue by spacekookie** [spacekookie/thunder#20]
This code

```rust
#![feature(proc_macro)]
extern crate thunder;
extern crate clap;

use thunder::thunderclap;

struct Loki;

#[thunderclap(example: String: "Error, here")]
impl Loki {
    fn hello() {
        println!("Hello, world!");
    }
}

fn main() {
    Loki::start();
}
```

will cause a program to panic when running the `hello` hook without a name, which should be optional!

This issue replaces #16

**Comment by gugahoa** [comment]
Just a little heads-up, the example in this one is for the issue #19 and vice-versa

**Comment by gugahoa** [comment]
This was fixed by #17

**Comment by spacekookie** [comment]
Ah yes, I thought I closed *something* manually yesterday :thinking:
